estate/models/estate_property.py

```diff
@@ -44,8 +44,12 @@ class EstateProperty(models.Model):
     property_type_id = fields.Many2one("property.type", string="Property Type")
     salesperson_id = fields.Many2one(
         "res.users",
-        string="Salesperson",
+        string="Salesman",
         default=lambda self: self.env.user.id,
     )
+    buyer_id = fields.Many2one("res.partner", string="Buyer", copy=False)
+    tag_ids = fields.Many2many("property.tag", string="Tags")
```

Comment:
strings not needed, remove them please

```suggestion
    buyer_id = fields.Many2one("res.partner", copy=False)
    tag_ids = fields.Many2many("property.tag")
```

Reply:
missing point here